stream/src/main/scala/org/apache/pekko/stream/impl/fusing/LazySingleSource.scala

```diff
@@ -0,0 +1,50 @@
+/*
```

Comment:
This header file is a slightly different variant compared to what else is in Pekko, see https://github.com/apache/incubator-pekko/blob/bd9ff4da5bb59234c39af8916fb7a06caa755052/project/AddMetaInfLicenseFiles.scala#L1-L16 for the correct version